advanced/Scripts/list.sh

```diff
@@ -47,13 +48,17 @@ helpFunc() {
 :::  -h, --help				Show this help dialog
 :::  -l, --list				Display your ${word}listed domains
 EOM
+if [[ "letter" == "b" ]]; then
```

Comment:
Will this need to be `"$letter"`? I think it's currently comparing the plain string `"letter"` to `"b"`. Not a bash expert though ;)